appyters/Patient_Cohorts_RNASeq_Viewer/templates/form.j2

```diff
@@ -17,3 +17,10 @@
 </div>
 {{ super() }}
 {% endblock %}
+
+
+{% block script %}
+{{ super() }}
+<script>window.MathJax = { MathML: { extensions: ["mml3.js", "content-mathml.js"]}}; console.log("hello");</script>
+<script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=MML_HTMLorMML"></script>
```

Comment:
Does this just fix your math in the markdown without additional triggering? I may want to add this to all appyters

Reply:
Yes - the MathML displayed fine without it on Safari but I had to add this script for proper display on Chrome.

Reply:
Never mind, it seems you're using it in your form options only.